backend/sirius-components-view-edit/src/main/java/org/eclipse/sirius/components/view/provider/FormDescriptionItemProvider.java

```diff
@@ -179,9 +179,11 @@ protected void collectNewChildDescriptors(Collection<Object> newChildDescriptors
         newChildDescriptors.add(this.createChildParameter(ViewPackage.Literals.FORM_DESCRIPTION__WIDGETS, radioDescription));
 
         BarChartDescription barChartDescription = ViewFactory.eINSTANCE.createBarChartDescription();
+        barChartDescription.setStyle(ViewFactory.eINSTANCE.createBarChartDescriptionStyle());
```

Comment:
Please also add this line into `FlexboxContainerItemProvider#collectNewChildDescriptors()`

Reply:
done